src/eggd_qc_to_workbooks.sh

```
@@ -63,9 +69,46 @@ main() {
     dx download -r "$path_to_reports_folder"
     cd /home/dnanexus/
 
+<<<<<<< HEAD
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)